readme.md

```diff
@@ -1,32 +1,86 @@
 * On Linux and Windows:
    * **Using Maven**: the simplest way to use the Azure IoT SDKs for Java to develop apps is to leverage Maven packages:
       * [Device SDK][device-maven]
       * [Service SDK][service-maven]
+   * **Clone the repository**: The repository is using [GitHub Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) for its dependencies. In order to automatically clone these submodules, you need to use the --recursive option as described here:
+   ```
+    git clone --recursive https://github.com/Azure/azure-iot-sdk-java.git  
+   ```
    * **Working with the SDKs code**: if you are working with the SDKs code to modify it or contribute changes, then you can clone the repository and build the libraries:
       * [Build Device SDK from code][device-code]
       * [Build Service SDK from code][service-code]
 * On Android: our Java device SDK can be used on Android using the API version 17 and higher:
    * [Device SDK][device-android]
 
+## Key features and roadmap
+
+### Device client SDK
+:heavy_check_mark: feature available  :heavy_multiplication_x: feature planned but not supported  :heavy_minus_sign: no support planned
+
+| Features                                                                                                         | mqtt                     | mqtt-ws                  | amqp                     | amqp-ws                  | https                    | Description                                                                                                                                                                                                                                                                                                                                 |
+|------------------------------------------------------------------------------------------------------------------|--------------------------|--------------------------|--------------------------|--------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Authentication](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-security-deployment)                     | :heavy_check_mark:       | :heavy_check_mark:*      | :heavy_check_mark:*      | :heavy_check_mark:*      | :heavy_check_mark:       | Connect your device to IoT Hub securely with supported authentication, including private key, SASToken, X-509 Self Signed and Certificate Authority (CA) Signed over MQTT.  *IoT Hub only supports X-509 CA Signed over AMQP and MQTT at the moment.  X509-CA Signed over AMQP is in progress.                                              |
```

Comment:
Incorrect about AMQP. I will send you email with details on X509 support. Please update this accordingly.

Reply:
Updated.  X509 over AMQP is not supported right?